src/modules/views/Drive/DriveFolderView.jsx

```diff
@@ -165,10 +166,12 @@ const DriveFolderView = () => {
     byDocId,
     isMobile,
     isNativeFileSharingAvailable,
-    shareFilesNative
+    shareFilesNative,
+    selectAll: () => selectAll(allResults.map(query => query.data).flat())
   }
   const actions = makeActions(
     [
+      selectAllItems,
```

Comment:
Adding selectAllItems here will make it appear everywhere there are actions involved, for example, when you right-click on an item:

For example, here:
```tsx
  <RightClickFileMenu
    key={file._id}
    doc={file}
    actions={actions}
  >
```

also

```jsx
<SelectionBar actions={actions} />
```

and 
```jsx
  <File
      key={file._id}
      attributes={file}
      withSelectionCheckbox
      withFilePath={withFilePath}
      thumbnailSizeBig={isBigThumbnail}
      actions={actions}
      refreshFolderContent={refreshFolderContent}
```

The better approach can be:
- give a context to the displayCondition action method
- or to separate the selectionBar actions from the file right click menu + file actions ( duplicates actions etc )